tests/test_arr_venue_v2.py

```diff
@@ -2213,6 +2229,10 @@ def test_resubmission_and_track_matching_data(self, client, openreview_client, h
 
         helpers.await_queue()
 
+        # Getting resubmissions should pass
+        previous_url_field = 'previous_URL'
+        resubmissions = openreview.arr.helpers.get_resubmissions(submissions, previous_url_field)
```

Comment:
using the code that you are testing in your test doesn't test anything :) 

Here in the test you can get the submissions that you know must be resubmissions, like `i == 6`